bin/rebuild-mixins-docs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# NOTE: This script should be executed from the root of the repository.
+
+tmp_doc_file="/tmp/rebuild-mixins-docs.tmp.md"
+mixins_dir="mixins"
+
+readme_lead='^<!-- BEGINNING OF TERRAFORM-MIXINS DOCS HOOK -->$'
```

Comment:
why using `BEGINNING` and `BEGIN`?

Reply:
`BEGINNING` is consistent with with the markers used by `terraform-docs` https://github.com/cloudposse/terraform-aws-components/blob/24806008e2d1b507b4e5d4a1d5ab490eea9774c1/modules/alb-controller/README.md?plain=1#L46

As for `BEGIN` — it's just natural I guess. Moreso than `BEGINNING OF DOC`,  at least for me.